EKF/control.cpp

```diff
@@ -377,7 +362,7 @@ void Ekf::controlExternalVisionFusion()
 				}
 			}
 
-			ev_vel_obs_var = matrix::max(ev_vel_var.diag(), sq(0.01f));
+			ev_vel_obs_var = matrix::max(getVisionVelocityVarianceInEkfFrame(), sq(0.05f));
```

Comment:
We mirror this change to the firmware side param lower bound. It is 0.01 right now, iirc.

Reply:
Don't think so. At the moment it is [0.1 m/s](https://github.com/PX4/Firmware/blob/master/src/modules/ekf2/ekf2_params.c#L747). This is a value that gave good results with test on the Realsense T265. 
That this value here is lower is okay, since it is a fixed lower bound that should guarantee stability of the filter. The other is a tuning parameter.

Reply:
I was referring to the lower bound in the metadata https://github.com/PX4/Firmware/blob/297e4dd5893128122109863ee79e30500e426553/src/modules/ekf2/ekf2_params.c#L743

Reply:
oh, good point.